### DISCUSSION TITLE
The code reference is not displayed correctly

### DISCUSSION THREAD

**Discussion by naosense** [community/community#43349] [Other Feature Feedback, Questions, & Ideas]
The code reference is not displayed correctly, just showing a link, not a specific code snippet.

![Screenshot_20230105_101309.jpg](https://user-images.githubusercontent.com/2225426/210686365-05aa671b-5954-4ec7-bebf-b030c9081b73.jpg)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @naosense ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
